tools/test/build_images.py

```diff
@@ -82,12 +93,19 @@ def build_images(matrix, script_args):
 
             args = ()
             env = {"DOCKER_BUILDKIT": "1"}
+            
+            # Use docker buildx build (or podman build if USE_PODMAN is set)
+            if use_podman:
+                docker_cmd = ['docker', 'build']
```

Comment:
Not sure, but should this be
```suggestion
                docker_cmd = ['podman', 'build']
```

Reply:
I think i missed this somehow (even in the comment i wrote `podman build`) Thanks.